intact-uniprot-export/src/test/java/uk/ac/ebi/intact/util/uniprotExport/writers/golinewriters/GOLineWriterVersion2Test.java

```diff
@@ -33,14 +39,37 @@ public void test_go_export(){
 
             writer.close();
 
-            File template = new File(GOLineWriterVersion1Test.class.getResource("/go_file_test2.txt").getFile());
+            File template = getTemplateFileWithCurrentDate();
```

Comment:
The expected file has the current date, which needs to be updated every time for the test to pass. I've updated the test to automatically set the date in the expected file to the current date when the test runs, so we don't need to manually update it.